bo/steuerbareressorce.go

```diff
@@ -0,0 +1,16 @@
+package bo
+
+import (
+	"github.com/hochfrequenz/go-bo4e/enum/steuerkanalsleistungsbeschreibung"
+)
+
+type Steuerbareressource struct {
+	Geschaeftsobjekt
+	SteuerbareRessourceId             string                                                               `json:"steuerbareRessourceId" validate:"required"` // Identifikationsnummer einer SteuerbareRessource
+	SteuerkanalsLeistungsbeschreibung *steuerkanalsleistungsbeschreibung.Steuerkanalsleistungsbeschreibung `json:"*steuerkanalsLeistungsbeschreibung"`        // Leistungsbeschreibung des Steuerkanals
+	ZugeordnetMSBCodeNr               string                                                               `json:"zugeordnetMSBCodeNr"`                       // Angabe des Messstellenbetreibers, der der Steuerbaren Ressource zugeordnet ist
```

Comment:
kannst du das bitte auch nullable machen ODER einen required tag adden? schau mal, wie wir das anderswo handhaben. in .net ist es auf jeden fall nullable.
https://github.com/Hochfrequenz/BO4E-dotnet/blob/d3d89270f94672c5502eea9a632b9de1d7b7ac12/BO4E/BO/SteuerbareRessource.cs#L52

deswegen würde ich vorschlagen, dass wir es hier auch nullable machen. sonst crasht die library wenn sie ein null an der stelle erhält.